src/status_im/contexts/wallet/send/select_collectible_amount/view.cljs

```diff
@@ -11,24 +12,20 @@
 (defn view
   []
   (let [on-close              (rn/use-callback #(rf/dispatch [:navigate-back]))
-        [value set-value]     (rn/use-state 1)
-        inc-value             (rn/use-callback (fn [] (set-value (inc value)))
-                                               [value])
-        dec-value             (rn/use-callback (fn [] (set-value (dec value)))
-                                               [value])
-        add-digit             (rn/use-callback (fn [digit]
-                                                 (set-value (+ (js/parseInt digit)
-                                                               (* value 10))))
-                                               [value])
-        delete-digit          (rn/use-callback (fn []
-                                                 (set-value (Math/floor (/ value 10))))
-                                               [value])
-
+        [value set-value]     (rn/use-state controlled-input/init-state)
         send-transaction-data (rf/sub [:wallet/wallet-send])
         collectible           (:collectible send-transaction-data)
         balance               (utils/collectible-balance collectible)
         preview-uri           (get-in collectible [:preview-url :uri])
-        incorrect-value?      (or (< value 1) (> value balance))]
+        incorrect-value?      (controlled-input/input-error value)]
+    (rn/use-mount
+     (fn []
+       (set-value #(controlled-input/set-numeric-value % 1))
+       (set-value #(controlled-input/set-lower-limit % 1))))
+    (rn/use-state
+     (fn []
+       (set-value #(controlled-input/set-upper-limit % balance)))
+     [balance])
```

Comment:
`use-state` doesn't have a two arity variant, and the return value is ignored.

Was this an attempt to use `use-effect`?
If this PR is working without properly using this, probably it was never needed or it's having a hidden bug :monocle_face:

Reply:
Good catch, thanks!
Yes, the idea was to use `use-effect`. 
I assume that there can be a case when collectible balance has changed from outside while we are still on sending screen. For this case I wanted to update state to reflect the possible change.
Wdyt about possibility of this scenario?

Reply:
Totally makes sense to cover this case @vkjr !

It may happen, I think the transaction will fail if we try to send more collectibles than available, but checking this is a good improvement in terms of UX :+1: